crates/sui-benchmark/tests/simtest.rs

```diff
@@ -103,7 +103,12 @@ mod test {
 
         let client_node = sui_simulator::runtime::NodeHandle::current().id();
         register_fail_points(
-            &["batch-write", "transaction-commit", "put-cf"],
+            &[
+                "batch-write",
+                "transaction-commit",
+                "put-cf",
+                "before-open-new-epoch-store",
```

Comment:
we might want to make this new fail point much higher likelyhood than the others. If you look below you can see we are failing after 1% of writes for the db-based fail points. But for this one, it is only called once, so the failure will only be injected once every 100 / num_validators epochs.

Reply:
I added another reconfig test which crashes one node with 100% probability every epoch change (10s) at the fail point for 120s test duration.